src/core.js

```diff
@@ -159,3 +149,8 @@ jQuery.fn.size = function() {
 	migrateWarn( "jQuery.fn.size() is deprecated; use the .length property" );
 	return this.length;
 };
+
+jQuery.parseJSON = function() {
+	migrateWarn( "jQuery.parseJSON is deprecated; use JSON.parse" );
```

Comment:
I understand that a lot of people have this problem & Migrate helps here but those people are not running jQuery 1.12/2.2 without a Migrate and I assume Migrate 3 only plans to targets those users. Even if they had this issue, they had to already have it resolved. Moving from jQuery 2.2 without Migrate to jQuery 3.0 with Migrate 3.0 that still contains this workaround would only introduce a potential for new bugs for them if they already have logic that handles invalid AJAX requests (such as ones with a JSON content type but empty body) that would suddenly start being valid.